### DISCUSSION TITLE
How I can configure storybook to work with styled components and unistyles

### DISCUSSION THREAD

**Discussion by gustavosorati** [storybookjs/react-native#572] [Q&A] [answered]
Hello everyone, this is my first contact with the storybook, and I would like to know how I can configure it to use it together with **unistyles** and **styled-components**, I am using expo in an existing project.

**Comment by dannyhw** [comment]
I don't think anything is needed to make unistyles work (other than whatever setup you would normally do for an app), for styled-components the only thing you might need is to provide a decorator with the theme provider if there is one. You can do that per story or globally via preview.tsx.

**Reply by gustavosorati** [reply]
in fact it was not necessary to do anything, everything works, but the type is not working and the imports do not work with the shortcuts configured in tsconfig.

For configuration I tried to do something like this, and apparently it worked, but I don't know if it's the right way, for shortcuts I haven't found it yet, could you also direct me to a search path?

```javascript
decorators: [
     withThemeFromJSXProvider({
       GlobalStyles: theme // my theme range
     })
   ]
```

**Reply by dannyhw** [reply]
Sorry im not sure what you are referring to could you explain

**Reply by gustavosorati** [reply, marked as answer]
I was using stories inside the storybook folder, from the moment I started using the stories files together with my components inside the app folder of my expo application, the errors I was having with typing and absolute paths disappeared.

So I'll mark it as complete, thanks for trying to help.
